### ISSUE TITLE
Named routes

### ISSUE THREAD

**Issue by cking** [husobee/vestigo#54]
Would it be possible to add names (or tags) to routes and maybe a route generator.

Instead of manually writing routes for redirects in my code, I'd prefer to call something like `router.Generate(name string, params map[string]string)`

**Comment by husobee** [comment]
Good idea.  Will investigate, unless you would like to PR this functionality.

**Comment by cking** [comment]
I am right now busy with my project, but I can start on it in a few days.
